tests/ui/transmutability/arrays/should_require_well_defined_layout.stderr

```diff
@@ -23,7 +23,7 @@ error[E0277]: `u128` cannot be safely transmuted into `[String; 0]` in the defin
   --> $DIR/should_require_well_defined_layout.rs:27:47
    |
 LL |         assert::is_maybe_transmutable::<u128, repr_rust>();
-   |                                               ^^^^^^^^^ `[String; 0]` does not have a well-specified layout
+   |                                               ^^^^^^^^^ `u128` does not have a well-specified layout
```

Comment:
what's going on here?

Reply:
I think the fact that we now reduce the `Answer` tree in a slightly different order, made it so that some errors are bubbling up instead of other ones, but I need to look into it more